canonical-web-and-design/ubuntu.com
# Add intel-nuc instructions /appliance/plex/intel-nuc

## Done

- Builts /appliance/plex/intel-nuc
- Updated the [copy doc](https://docs.google.com/document/d/1gb03YbX5ZQ_oO74M36b3ybCPwqq29d8Q17c6KaguW18/edit#)
- Refactored the pi includes

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/appliance/plex/intel-nuc
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that the http://0.0.0.0:8001/appliance/plex/raspberry-pi page works as well.

Fixes #7494
